constitution.tex

```diff
@@ -738,7 +738,7 @@
 \asubsubsubsection{Changes}
 The following process is used for making changes to the Code of Conduct document.
 \begin{enumerate}
-	\item A Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
+	\item An Active Member drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
```

Comment:
```suggestion
	\item An Active Member or Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
```

Reply:
I see where you are coming from as a non-active RTP, however I feel that matters regarding the COC are primarily the concern of active members. Once an RTP leaves Active Member status they shouldn't concern themselves with the COC.

Reply:
This suggests that RTPs who were not active members have not been involved in day to day matters, which is simply not the case in recent memory.

Reply:
Reminder that not active != not at RIT

Reply:
IIRC Non-Active RTPs can't vote on Constitutional Amendments, and modifying the COC is of equal stature as the Constitution.

Reply:
Also, RTPs do not need to be active when they become RTPs